<launch>
  <node pkg="firefly_mapping" type="gcs_mapping" name="gcs_mapping" output="screen"/>
  <node pkg="firefly_telemetry" type="gcs_telemetry.py" name="gcs_telemetry" output="screen"/>
  <node pkg="firefly_mapping" type="gps_to_local.py" name="gps_to_local" output="screen"/>
  <node pkg="firefly_mapping" type="mapping_accuracy" name="mapping_accuracy" output="screen"/>
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find firefly_mapping)/rviz/gcs.rviz" />
</launch>